underdocs-version/pom.xml

```diff
@@ -0,0 +1,70 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>underdocs</groupId>
+        <artifactId>underdocs-parent</artifactId>
+        <version>1.0.1</version>
+    </parent>
+
+    <artifactId>underdocs-version</artifactId>
+
+    <name>Underdocs Version</name>
+    <description>Version information collecting component of the Underdocs documentation gerator tool.</description>
```

Comment:
Typo here

Reply:
Didn't notice that, but now looking at other POM files they had these typos as well. Now they are fixed.

Reply:
Nice! 💯